test/performance/perf_test.py

```diff
@@ -44,7 +45,7 @@
 
 OUTPUT_FILE = os.environ.get('OUTPUT_FILE')
 
-result_data = []
+result_data: List[Any] = []
```

Comment:
Isn't this annotation redundant?

Reply:
`test/performance/perf_test.py:48: error: Need type annotation for "result_data" (hint: "result_data: List[<type>] = ...")`

Reply:
Oh that's surprising. I thought it would default to List. TIL.

Reply:
It defaulted to `List[unknown]`, but adding `:  List` allowed it to reference as `List[Any]`, so the `[Any]` was redundant.